Tyro/Error.swift

```diff
@@ -8,26 +8,26 @@
 
 import Foundation
 
-public protocol JSONErrorType : ErrorType {}
+public protocol JSONErrorType : Error {}
 
 public enum JSONError : JSONErrorType {
     case Array([JSONError])           // array of errors
     case TypeMismatch(String, String) // expected / actual
-    case Error(ErrorType, String)     // error / message
+    case Error(Error, String)     // error / message
```

Comment:
You mean lowercase enum cases?

I saw that in Swiftx (the swift 3 version) Either still has uppercase members for left and right.  I though since these really wanted to indicate types somehow (Array, Object, String ...) the exception would be acceptable. Looked a little odd when I tried. Will do anyways.